app/ts/Workspace/WorkspaceContext.tsx

```diff
@@ -67,20 +74,11 @@ export function WorkspaceContextProviderComponent({
   }, []);
 
   useEffect(() => {
-    if (scales && oldWorkspace) {
+    if (hasScalevalues(scales) && oldWorkspace) {
       setObservedRanges(calculateObservedRanges(scales, workspace));
```

Comment:
interesting that it checks for oldworkspace but then uses `workspace` - i wonder if there's a race condition here

Reply:
Don't know, haven't looked in to it, as it wasn't part of this bug.